auditbeat/magefile.go

```diff
@@ -169,29 +164,8 @@ func GoUnitTest(ctx context.Context) error {
 	return devtools.GoTest(ctx, devtools.DefaultGoTestUnitArgs())
 }
 
-// GoIntegTest executes the Go integration tests.
-// Use TEST_COVERAGE=true to enable code coverage profiling.
-// Use RACE_DETECTOR=true to enable the race detector.
-func GoIntegTest(ctx context.Context) error {
-	mg.Deps(Fields)
```

Comment:
Some integration tests we are removing in this PR depended on `Fields`, are we keeping these dependencies or they were not needed?

Reply:
Hmm, you need to give me more references. I can confirm that all tests passed in the Beats CI.